packages/loot-core/src/client/selectors.ts

```diff
@@ -10,8 +10,5 @@ const getPrefsState = createSelector(getState, state => state.prefs);
 const getLocalPrefsState = createSelector(getPrefsState, prefs => prefs.local);
 
 export const selectNumberFormat = createSelector(getLocalPrefsState, prefs =>
-  getNumberFormat({
-    format: prefs.numberFormat,
-    hideFraction: prefs.hideFraction,
-  }),
+  getNumberFormat(2, prefs.numberFormat, prefs.hideFraction),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'd actually recommend keeping the previous implementation here (i.e. an object as the argument). It allows to expand this argument list as we see fit without causing too much bloat. It also allows to do stuff like `getNumberFormat({ hideFraction: true })` as opposed to `getNumberFormat(undefined, undefined, true)`

Reply:
Yea, good call. I totally overlooked that.